src/index.js

```diff
@@ -8,6 +8,8 @@ const gutenbergSetup = () => {
 	const apiFetch = require( '@wordpress/api-fetch' ).default;
 	const wpData = require( '@wordpress/data' );
 
+	I18nManager.forceRTL( false ); // Change to `true` to debug RTL layout easily.
```

Comment:
Not sure if we should leave this here. But for the moment it could be useful for fast checking RTL layout in both iOS and Android, without the need of `AndroidStudio` or `Xcode` or changing settings on device.

Reply:
I guess we can leave it here. It doesn't hurt, no?